### PR TITLE
fixed variant PropTypes is undefined and SeriesNumber type is Number bug

### PR DESCRIPTION
**Pull request by 027xiguapi**
…mber bug

### PR Checklist

- [ ] Brief description of changes
- [ ] Links to any relevant issues
- [ ] Required status checks are passing
- [ ] User cases if changes impact the user's experience
- [ ] `@mention` a maintainer to request a review

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

File: platform/viewer/public/config/default.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ window.config = {
     {
       friendlyName: 'dcmjs DICOMWeb Server',
       namespace: '@ohif/extension-default.dataSourcesModule.dicomweb',
-      sourceName: 'dicomweb',
+      sourceName: 'dicomweb', // 接口设置
```
Comment (sedghi) [inline platform/viewer/public/config/default.js:12]: can you remove this please?

**Review by sedghi — CHANGES_REQUESTED** [review]
thanks for your PR, please see my comment

**Comment by 027xiguapi** [comment]
ok，I have solved it

**Comment by sedghi** [comment]
Can you rebase please?

**Comment by 027xiguapi** [comment]
Yes, check it

**Comment by sedghi** [comment]
Hey, thanks for updating it. You can change your based to master since we merged v3-stable to master. Also you need to pull in new updates since we changed the signiture of buttons

**Pull request closed without merging**